libs/file-storage/src/lib/controller/abstract-file-storage.controller.ts

```diff
@@ -19,11 +27,11 @@ export abstract class AbstractFileStorageController {
       If the file does not exist, an error will occur only when an attempt is made to download the file using the pre-signed URL.
     `,
   })
-  @Post('download-file')
+  @Get('download-file')
   @HttpCode(HttpStatus.MOVED_PERMANENTLY)
   protected async downloadFileFromAWS(
     @Res() reply: FastifyReply,
-    @Body() fileDownloadRequest: FileDownloadRequest,
+    @Query() fileDownloadRequest: FileDownloadRequest,
```

Comment:
it's a little bit strange to keep it as query param, it definitely should be a pathparam